django_cloud_deploy/tests/cleanup/resource_cleanup.py

```diff
@@ -0,0 +1,113 @@
+            return False
+        now = datetime.datetime.utcnow()
+        create_time = utils.parse_creation_time(resource_name)
+        diff = now - create_time
+        return diff > self.MAX_DIFF
+
+    def delete_expired_clusters(self):
+        container_service = discovery.build(
+            'container',
+            'v1',
+            credentials=self.credentials,
+            cache_discovery=False)
+        for cluster_name in self.list_clusters(container_service):
+            if self._should_delete(cluster_name):
+                self._delete_cluster(cluster_name, container_service)
+
+    def delete_expired_buckets(self):
+        storage_service = discovery.build(
+            'storage',
+            'v1',
+            credentials=self.credentials,
+            cache_discovery=False)
+        for bucket_name in self.list_buckets(storage_service):
+            if self._should_delete(bucket_name):
+                self._delete_bucket(bucket_name, storage_service)
+
+    def delete_expired_sql_instances(self):
+        sqladmin_service = discovery.build(
+            'sqladmin',
+            'v1beta4',
+            credentials=self.credentials,
+            cache_discovery=False)
+        for instance_name in self.list_instances(sqladmin_service):
+            if self._should_delete(instance_name):
+                self._clean_up_sql_instance(instance_name, sqladmin_service)
+
+    def delete_expired_service_accounts(self):
+        iam_service = discovery.build(
+            'iam', 'v1', credentials=self.credentials, cache_discovery=False)
+        for account_email in self.list_service_accounts(iam_service):
+            account_name = account_email.split('@')[0]
+            if self._should_delete(account_name):
+                self._delete_service_account(account_email, iam_service)
+
+    def reset_expired_iam_policy(self):
+        cloudresourcemanager_service = discovery.build(
+            'cloudresourcemanager',
+            'v1',
+            credentials=self.credentials,
+            cache_discovery=False)
+        request = cloudresourcemanager_service.projects().getIamPolicy(
+            resource=self.project_id)
+        policy = request.execute()
+        for binding in policy['bindings']:
+            for member in binding['members']:
+                # Member should look like the follows:
+                # serviceAccount:sa-20190301000243-bu47@aaa.com
+                service_account_name = member.split('@')[0].split(':')[-1]
+                if self._should_delete(service_account_name):
+                    binding['members'].remove(member)
+
+        policy['bindings'] = [b for b in policy['bindings'] if b['members']]
+        body = {'policy': policy}
+        request = cloudresourcemanager_service.projects().setIamPolicy(
+            resource=self.project_id, body=body)
+        request.execute()
+
+    def test_resource_cleanup(self):
```

Comment:
```suggestion
    def cleanup_test_resources(self):
```

Reply:
Actually this whole class inherits unittest.TestCase indirectly. So to run this function, I can either make the function name starting with "test", or manually register this test case. Now I just manually register this test case.

Reply:
I tried manually registering this test case and run it with pytest, looks like pytest cannot find it. I also tried run it with a script, for example, "python3 tests/cleanup/resource_cleanup.py", but nox will complain about not able to find module django-cloud-deploy. So what could be a good way to automatically run this script?